CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 ########################################
 # General setup
 #
-cmake_minimum_required(VERSION 2.8.8)
+cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR)
```

Comment:
Can you remove the FATAL_ERROR? It's for very old CMake builds, no one is putting it anymore: https://cmake.org/cmake/help/v3.7/command/cmake_minimum_required.html